www/~/%username/subscriptions.json.spt

```diff
@@ -0,0 +1,100 @@
+""" Get or change authenticated user's subscriptions.
+"""
+from aspen import Response
+from gratipay.models.team import Team
+from gratipay.utils import get_participant
+
+[-----------------------------------------------------------------------------]
+
+def format_subscription(s):
+    return {
+        "team_name": s.team_name,
+        "team_slug": s.team_slug,
+           "amount": str(s.amount),
+              "due": str(s.due),
+            "ctime": s.ctime,
+            "mtime": s.mtime
```

Comment:
Interesting! We've gone back and forth over [leading-comma](https://gist.github.com/isaacs/357981) style [before](https://botbot.me/freenode/gratipay/2014-01-13/?msg=9719489&page=4), but this style is new to me. Our SQL [formatting](https://github.com/gratipay/gratipay.com/pull/3896#discussion_r50124769) has inspired you, I take it? :-)